extensions/auto-html.js

```diff
@@ -7,6 +8,9 @@ var linkifyUrls = function(text) {
 var imagifyLinks = function(text) {
     return text.replace(imageLinkRegex, "$1<img src='$2' /></a>");
 };
+var roomifyLinks = function(text) {
+  return text.replace(roomMeRegex, "$1<a href='/rooms/$1'>#$2</a>");
```

Comment:
I think this should be $1<a href='/rooms/$2'>#$2</a> otherwise you'll link to the wrong room